fe/estimator_abfe.py

```diff
@@ -132,6 +132,10 @@ def simulate(lamb, box, x0, v0, final_potentials, integrator, barostat, equil_st
     equil_schedule = np.ones(equil_steps)*lamb
     ctxt.multiple_steps(equil_schedule)
 
+    # (ytz): intentionally hard-coded, I'd rather the end-user *not*
```

Comment:
(note that equilibration will still use whatever default barostat interval is used, but it's assumed that once we're at equilibrium, interval=25 should be sufficient).